Healthy/Classes/Entities/MealDetails.swift

```diff
@@ -0,0 +1,6 @@
+// MARK: - Meal Details
+struct MealDetails {
+    let id, name, drinkAlternate, category, area, instructions, thumbnail, tags: String
+    var ingredients: [String] = []
+    var measures: [String] = []
```

Comment:
```suggestion
    let ingredients: [String] = []
    let measures: [String] = []
```

Mutability is a big source of bugs. Avoid it.